R/CreatePackageVignette.R

```diff
@@ -0,0 +1,101 @@
+#' @title pkgnet Report as Vignette
+#' @name CreatePackageVignette
+#' @description Create pkgnet package report as an Rmarkdown vignette. This
+#'    vignette is able to be rendered with the
+#'    \code{\link[knitr:vignette_engines]{knitr::rmarkdown}} vignette engine
+#'    into HTML vignettes upon package building. It is also compatible with
+#'    \code{\link[pkgdown:build_articles]{pkgdown}} sites.
+#' @param pkg_name (string) name of a package
+#' @param pkg_path (string) The path to the package repository. If given, coverage
+#'                 will be calculated for each function. \code{pkg_path} can be an
+#'                 absolute or relative path.
+#' @param pkg_reporters (list) a list of initialized package reporters
+#' @param vignette_path (string) The path and filename of the output vignette
+#'    file. The default assumes your working directory is the package root.
+#' @importFrom rlang enexpr
+#' @importFrom assertthat assert_that is.string is.readable
+#' @importFrom tools file_ext
+#' @importFrom R6 is.R6Class
+#' @importFrom glue glue
+#' @export
+CreatePackageVignette <- function(pkg_name
```

Comment:
@jayqi I think a working example and/or an output message might go a long way here.  I could see someone running this function in the terminal and wondering where the output is located and if the function worked.

Reply:
Yeah, I was actually thinking about writing... <sup>wait for it...</sup> a vignette about how to use this. I think if we get your gallery working, we could just make it a general vignette about how to publish and share pkgnet reports.

Reply:
@jayqi , I think that may be better suited for the current Articles section of the website, but I get what you mean.

Reply:
Looking forward to seeing it.